src/app/teacher/teacher.service.ts

```diff
@@ -251,12 +251,13 @@ export class TeacherService {
     return this.http.post<any>(this.forgotPasswordUrl, params, { headers: headers });
   }
 
-  getVerificationCodeEmail(username) {
+  getVerificationCodeEmail(username, token) {
     const headers = new HttpHeaders({ 'Cache-Control': 'no-cache' });
-    const params = new HttpParams().set('username', username);
+    let params = new HttpParams();
+    params = params.set('username', username);
+    params = params.set('token', token);
     return this.http.get<any>(this.getVerificationCodeUrl, { headers: headers, params: params });
   }
-
```

Comment:
I think you can undo these changes in this file because they are unrelated to this issue. It looks like these changes were made for adding recaptcha to the forgot teacher password form.